src/booking/containers/utils/date-validators.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { FormBuilder, FormControl } from '@angular/forms';
+import {
+  BookingDateFormErrorMessage,
+  TimeOption,
+} from '../../models';
+import { DateValidators } from './date-validators';
+
+describe('DateValidators', () => {
+  let control: FormControl;
+
+  beforeEach(() => {
+    control = new FormControl({});
+  });
+
+  it('dayValidator invalid case', () => {
```

Comment:
The descriptions should be changed to make them read better, e.g. "should have an invalid date" and "should not allow the start date to be after the end date"

Reply:
Descriptions changed to resolving conversation.